lib/macho/load_commands.rb

```diff
@@ -52,8 +52,10 @@ module MachO
     0x2b => :LC_DYLIB_CODE_SIGN_DRS,
     0x2c => :LC_ENCRYPTION_INFO_64,
     0x2d => :LC_LINKER_OPTION,
-    0x2e => :LC_LINKER_OPTIMIZATION_HINT
-  }
+    0x2e => :LC_LINKER_OPTIMIZATION_HINT,
+    0x2f => :LC_VERSION_MIN_TVOS,
+    0x30 => :LC_VERSION_MIN_WATCHOS
```

Comment:
Style nit: Always use a comma here (and in other hashes/arrays) to terminate the last element. Makes both the diff cleaner if things are added in the future and makes reordering elements less error-prone (because the last element is no different from every other element).